hack/make/test-unit

```diff
@@ -48,8 +48,8 @@ bundle_test_unit() {
 			| grep -v github.com/docker/docker/man \
 			| grep -v github.com/docker/docker/integration-cli)
 	fi
-
 	go test -cover -ldflags "$LDFLAGS" "${BUILDFLAGS[@]}" $TESTFLAGS $pkg_list
+	go test -cover -ldflags "$LDFLAGS" "${BUILDFLAGS[@]}" $TESTFLAGS github.com/docker/docker/pkg/loopback -test.root
```

Comment:
Other tests should have been already failing under non-root, but if you want to skip your test on non-root, I suggest checking UID as in https://github.com/moby/moby/blob/master/pkg/mount/mounter_linux_test.go#L14